### ISSUE TITLE
When we create an account, we should be added to our mailing list

### ISSUE THREAD

**Issue by jonquet** [agroportal/project-management#27]
Quand on crée un compte il y a une checkbox qui offre de s'inscrire a une mailing liste de BioPortal... ce qui ne se passe pas d'ailleurs.

Faudrait que chez nous cela implique une inscription a agroportal-users@lirmm.fr 
(que l'on peut faire a la main, si on a une notification qui dit que qq vient de creer un compte).
Je peux faire les inscritpions (j'ai les mains sur la mailing liste) maunellement;

**Comment by jonquet** [comment]
Je re-ouvre ce ticket pour qu'on essaye avec @syphax-bouazzouni de l'autmatiser du coup.

**Comment by jonquet** [comment]
(translation of the tracker in English) 

When you create an account there is a checkbox that offers to subscribe to a BioPortal mailing list... 
<img width="350" alt="image" src="https://user-images.githubusercontent.com/7379183/198667197-0f8be561-26c4-4543-b982-48fc89ae9469.png">

With AgroPortal, this would have to involve registration at agr**ortal-users@lirmm.fr
We had mentioned making an automatic call to Sympa mailing list server to add a member to the mailing list.

**Comment by jonquet** [comment]
@jvendetti @mdorf Does this feature work on BioPortal side ?

**Comment by jvendetti** [comment]
No, it hasn't worked for a long time: https://github.com/ncbo/bioportal_web_ui/issues/60

**Comment by syphax-bouazzouni** [comment]
The implementation of the feature depends on the Mailing List Management Software used, but generally, it consists of sending emails with **a specific subject** to the server mail.

In our case (LIRMM/UM), our used tool is [sympa](https://www.sympa.community/), and to subscribe a user to a mailing list we need to send a mail to sympa@lirmm.fr with the subject `subscribe list_name lastname firstname` (see documentation: https://listes.lirmm.fr/sympa/help/user-subscribe.html)

In the case of Stanford, the tool used is [Mailman](https://listes.umontpellier.fr/sympa/help/user-subscribe.html), and the subscription process is different, with this tool you need to send a mail from the user mail to the list subscription address, which will be in the form LISTNAME-join@DOMAIN. see https://www.list.org/mailman-member/node13.html

In the code, this was implemented in the UI, here (unchanged since 2011)
https://github.com/ncbo/bioportal_web_ui/blob/master/app/models/notifier.rb#L41

**Comment by jonquet** [comment]
An idea tonight ... for our button that are sendin an email maybe it could be even betetr to redirect to 
https://listes.lirmm.fr/sympa/info/agroportal-users

Then someone can decide by itself to register or un register.

**Comment by jonquet** [comment]
We might restart this thread using another strategy (pull from the SYMPA serveur).

**Comment by imadbourouche** [comment]
should work on this in 2026
